tests/functional/TestArbiterVolume/config/heketi.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
As a commit wide comment, I am not sure if I would want to add one more suite here. Mainly because I would like the infra code to be the same and am already working on a patch to merge 4 Vagrant based suites we have. Do we had a valid reason for new suite?

Reply:
I was a bit on the fence about that as well. I figured in the worst case it would be easier to merge things back together rather than split them apart. Plus, heketi_test.go feels a bit crammed full of stuff. However, we can still consolidate the suites but keep separate .go files. Does that sound like a reasonable compromise?

Reply:
Yes, for now if you could rename the test file to arbiter_test.go or similar and add it to smoke test it would be sufficient.